src/main/java/core/basesyntax/service/TransactionParser.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.OperationType;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionParser {
+    public List<FruitTransaction> parseTransactions(List<String> lines) {
```

Comment:
Don't begin method implementation with an empty line. Please remove the empty line before the start of the method body to adhere to the coding standards.